cli/connect/shortcuts.go

```diff
@@ -0,0 +1,26 @@
+package connect
+
+// getShortcutsList is a command for a get shortcuts and hotkeys list.
+const getShortcutsList = "\\shortcuts"
+
+const shortcutListText = `---
+- - |
+    Available hotkeys and shortcuts:
+
+       Ctrl + J / Ctrl + M [Enter] -- Enter the command
+       Ctrl + A [Home]             -- Go to the beginning of the line
+       Ctrl + E [End]              -- Go to the end of the line
+       Ctrl + P [Up Arrow]         -- Previous command
+       Ctrl + N [Down Arrow]       -- Next command
+       Ctrl + F [Right Arrow]      -- Forward one character
+       Ctrl + B [Left Arrow]       -- Backward one character
+       Ctrl + H [Backspace]        -- Delete character before the cursor
+       Ctrl + I [Tab]              -- Get next completion
+       BackTab                     -- Get previous completion
+       Ctrl + D                    -- Delete character under the cursor
+       Ctrl + W                    -- Cut the word before the cursor to the clipboard
+       Ctrl + K                    -- Cut the line after the cursor to the clipboard
+       Ctrl + U                    -- Cut the line before the cursor to the clipboard
+       Ctrl + L                    -- Clear the screen
+       Ctrl + R                    -- Enter in the reverse search mode
+...`
```

Comment:
Please, take a look at https://github.com/tarantool/tt/blob/master/cli/connect/console.go#L330-L355

Reply:
fixed

Reply:
And also at keybinds: https://github.com/tarantool/tt/blob/master/cli/connect/console.go#L330-L341. It seems that not everything has been added.

Reply:
fixed